artemis/src/main/java/tech/pegasys/artemis/cli/options/P2POptions.java

```diff
@@ -36,7 +36,7 @@
   public static final int DEFAULT_P2P_PORT = 30303;
   public static final boolean DEFAULT_P2P_DISCOVERY_ENABLED = true;
   public static final List<String> DEFAULT_P2P_DISCOVERY_BOOTNODES =
```

Comment:
This is pretty subtle / easy to break - we should probably add a test that checks the right values are set based on CLI args to  `BeaconNodeCommandTest`.

Reply:
Good idea. Added.